index.js

```diff
@@ -52,16 +52,21 @@ function compressPlugin (fastify, opts, next) {
 
   // add onSend hook onto each route as needed
   fastify.addHook('onRoute', (routeOptions) => {
-    // Manage compression options
+    // If route config.compress has been set it takes precedence over compress
```

Comment:
@mcollina > I made the change, but I don't know if I misunderstood :rofl: so right now `config.(de)compress` takes prevalence over `(de)compress` options, if it should be the over way around I will make the needed changes :smile: